set.mm

```diff
@@ -522553,6 +522614,12 @@ Real and complex numbers (cont.)
 #*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
 $)
 
+  $( Set up a new wff var $)
+  $v al $.  $( Greek alpha $)
+
+  $( Let variable ` al ` be a wff. $)
+  walpha $f wff al $.
+
```

Comment:
I wonder in which order Norm chose to introduce wff variable names. It seems to somehow start from the end of the greek alphabet and go backwards, with some jumps. I'm not sure what would be the next letter to choose.